### PR TITLE
PR_VM_Test&Recordings_Mar18

### PR DESCRIPTION
**Pull request by HimanshuBri**
New Test Cases and Recordings

### CODE REVIEW

File: test/commands/cli.vm.create_docker-tests.js [hunk 1]
```diff
@@ -209,7 +209,7 @@ describe('cli', function() {
 
       it('Create Docker VM with invalid docker cert dir should throw error', function(done) {
         getImageName('Linux', function(ImageName) {
-          var cmd = util.format('vm docker create %s %s %s %s --json --ssh 22 --docker-cert-dir D:/foo/bar',
+          var cmd = util.format('vm docker create %s %s %s %s --json --ssh 22 --docker-cert-dir F:/foo/bar',
```
Comment (babumuralidharan) [inline test/commands/cli.vm.create_docker-tests.js]: @HimanshuBri, @Sreekanthbs why did we change the docker cert dir here ??

File: test/commands/cli.vm.staticvm_create-from-tests.js [hunk 2]
```diff
@@ -100,6 +100,9 @@ describe('cli', function() {
 
       it('VM Create-from a file and assert the static ip', function(done) {
         var Fileresult = fs.readFileSync(file, 'utf8');
+		var ioType = '\"iOType\":';
```
Comment (babumuralidharan) [inline test/commands/cli.vm.staticvm_create-from-tests.js]: what npm package is this iOType depended ON ? Is it fs?

Comment (babumuralidharan) [inline test/commands/cli.vm.staticvm_create-from-tests.js]: Also we need to include the excluded test cases in the testlist.txt which is missing from this PR. Can you update and send that as well ?

File: test/commands/cli.vm.create_docker-tests.js (hunk 1 above)
Comment (HimanshuBri) [inline test/commands/cli.vm.create_docker-tests.js]: @babumuralidharan we changed it for some testing purpose, will revert back it to original one.

Comment (HimanshuBri) [inline test/commands/cli.vm.create_docker-tests.js]: @babumuralidharan we changed it for some testing purose, will revert it to original one.

File: test/commands/cli.vm.staticvm_create-from-tests.js (hunk 2 above)
Comment (sreekanthbs) [inline test/commands/cli.vm.staticvm_create-from-tests.js]: @babumuralidharan here iOType is dependent on fs.

Comment (HimanshuBri) [inline test/commands/cli.vm.staticvm_create-from-tests.js]: @babumuralidharan there were 8 files which were excluded in testlist.txt which we have already merged in this PR. 
Those files are 
1) cli.vm.create_win_rdp-tests.js
2)cli.vm.disk.attachDetach-tests.js
3)cli.vm.disk.attachnew-tests.js
4)cli.vm.export_create_from-tests.js
5)cli.vm.create_staticvm-tests.js
6)cli.vm.staticvm_create-from-tests.js
7)cli.vm.staticvm_docker-tests.js
8)cli.vm.create_docker-tests.js

**Pull request merged**